checkout-service/database/seed/dataGenerator.js

```diff
@@ -0,0 +1,51 @@
+const Faker = require('faker');
+
+
+const CreateFakeData = () => {
+
+  // centsForPrice is biased to .99 because of its popularity in prices
+  let centsForPrice = [.49, .75, .85, .89, .99, .99];
```

Comment:
Is centsforPrice defined to be used in amount so that amount us in decimal format ?

Reply:
Yeah I wanted to add realistic looking prices. Faker returned numbers with 0 cents (e.g. 650.00). So I went ahead and created the array so product prices have those cents (e.g. 65.99)